Arrowgene.Ddon.Shared/Model/ClientItemInfo.cs

```diff
@@ -68,6 +68,19 @@ public StorageType StorageType
             }
         }
 
+        public EquipSlot? EquipSlot { get
+            {
+                if (SubCategory is null) return null;
+                else if (SubCategory > ItemSubCategory.JewelrySubCategoryOffset) return Model.EquipSlot.Jewelry1; //All Jewelry
```

Comment:
Should there be a `Model` appended to equipslot? or is this a namespace issue? I think I also need this when trying to equip from treasure chests :)